dipy/io/vtk.py

```diff
@@ -15,8 +13,37 @@
     major_version = vtk.vtkVersion.GetVTKMajorVersion()
 
 
+def set_input(vtk_object, inp):
```

Comment:
@skoudoro remind me something this function stays in DIPY because we want to have surface loading support? Or there is some other reason? I thought the idea was to remove direct dependencies to vtk. And only indirectly depend through FURY?